.github/workflows/deploy.yml

```diff
@@ -99,7 +126,12 @@ jobs:
           $SLACK_NOTIFICATION_CI_RUNS
       - name: Set DOCKER_TAG
         run: |
-          echo "DOCKER_TAG=${REPOSITORY_URL}/emrit-${ENV}-${SERVICE}:${BUILD_NUMBER}" >> $GITHUB_ENV
+          if [[ $ENV == "prod" ]]
+          then
+            echo "DOCKER_TAG=${REPOSITORY_URL}/emrit-production-${SERVICE}:${TAG_TO_DEPLOY}" >> $GITHUB_ENV
```

Comment:
Where does the discrepancy between prod and production occur?

Reply:
In AWS ECR for prod, services use the template `emrit-production-[service]`, but `ENV === 'prod'`